OWML.Launcher/Program.cs

```diff
@@ -12,19 +15,46 @@ public class Program
         static void Main(string[] args)
         {
             var owmlConfig = GetOwmlConfig() ?? CreateOwmlConfig();
+            SaveConsolePort(owmlConfig);
             SaveOwmlPath(owmlConfig);
             var owmlManifest = GetOwmlManifest();
             var writer = OutputFactory.CreateOutput(owmlConfig, null, owmlManifest);
             var modFinder = new ModFinder(owmlConfig, writer);
-            var outputListener = new OutputListener(owmlConfig);
             var pathFinder = new PathFinder(owmlConfig, writer);
             var owPatcher = new OWPatcher(owmlConfig, writer);
             var vrPatcher = new VRPatcher(owmlConfig, writer);
             var app = new App(owmlConfig, owmlManifest, writer, modFinder,
-                outputListener, pathFinder, owPatcher, vrPatcher);
+                pathFinder, owPatcher, vrPatcher);
             app.Run(args);
         }
 
+        private static void SaveConsolePort(IOwmlConfig owmlConfig)
+        {
```

Comment:
A lot of low level code in this method. Can some of it be extracted to a class dealing with socket/port stuff?